plugins/snowflake/setup.py

```diff
@@ -47,11 +47,8 @@
     },
     install_requires=[
         'dbt-core=={}'.format(package_version),
-        'snowflake-connector-python[secure-local-storage]==2.3.6',
-        'azure-common<2.0.0',
-        'azure-storage-blob>=12.0.0,<13.0.0',
-        'urllib3>=1.20,<1.26.0',
-        # this seems sufficiently broad
+        'snowflake-connector-python[secure-local-storage]~=2.4.1',
```

Comment:
How bad of an idea is this?

This is the equivalent to `>=2.4.1, ==2.4.*` (https://www.python.org/dev/peps/pep-0440/#compatible-release)

Reply:
Just for my clarity: This is effectively the same as `>=2.4.1, <2.5.0`, right?

I'm in favor of giving it a go. The SCP folks seem like they've gotten their act together, and the connector is in much better order than it was just a few months ago. Let's do this now, and keep a close eye on SCP releases between now and v0.20.0 final release :)

Reply:
Yep! Cool, my thinking exactly 👍